src/token-manager/index.ts

```diff
@@ -124,23 +124,17 @@ abstract class TokenManager {
     try {
       let parsedUrl;
 
-      // Check expected browser support
-      if (support === 'legacy' || support === undefined) {
-        // To support legacy browsers, iframe works best with short timeout
-        parsedUrl = new URL(await OAuth2Client.getAuthorizeUrl(authorizeUrlOptions));
-      } else {
```

Comment:
It's important that this is kept. We still need to be able to support the legacy `iframe` method for calling the `authorize` endpoint. So, that means the functionality of `getAuthorizeUrl` needs to be preserved, but the name is what's replaced with a more semantic name.